pom.xml

```diff
@@ -133,6 +134,7 @@
     <commons-cli.version>1.4</commons-cli.version>
     <commons-collections.version>4.4</commons-collections.version>
     <error_prone_annotations.version>2.3.4</error_prone_annotations.version>
+    <jaegertracing.version>0.34.2</jaegertracing.version>
```

Comment:
Looks like some new releases have come out since we started. Should we update to latest?

Reply:
Yes we should update.

Reply:
Upated in the new commit.